tests/contrib/test_gce.py

```diff
@@ -73,6 +78,8 @@ def _refresh_success_helper(self, bytes_response=False):
         self.assertEquals(None, credentials.access_token)
         credentials.refresh(http)
         self.assertEquals(access_token, credentials.access_token)
+        self.assertFalse(credentials.access_token_expired)
+        self.assertTrue(credentials.token_expiry > datetime.utcnow())
```

Comment:
Yep, certainly...this is an older habit for me to prevent myself from accidentally returning a mocked value that is not the right type: `utcnow` actually returns a `datetime.datetime` but I had to check the stdlib docs because it seemed possible it would return a Unix time `int`. Happy to update if you'd prefer the mock, though!